packages/contracts/test/dollar/BondingCurve.t.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
```

Comment:
it feels empty ...
can we add some tests to make sure that for different collateral deposit we have accurate tokens returned (using  different connecteor weight and baseY)